### PR TITLE
Feature/add viewmodels to controllers

### PR DESCRIPTION
**Pull request by Gh0stF1ghter**

### CODE REVIEW

**Review by MaliyOna — APPROVED** [review]

File: BLL/ViewModels/ClientViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+namespace BLL.ViewModels;
+
+public record ClientViewModel(
+    Guid Id,
+    string FullName,
+    string Email,
+    string PhoneNumber,
+    Vehicle? Vehicle,
+    IEnumerable<VehicleClientHistory> VehicleClientHistories
+    );
```
Comment (5Ka-me) [inline BLL/ViewModels/ClientViewModel.cs:12]: ViewModels should be in API layer.
API - ViewModels
BLL - Models
DAL - Entities
In your case "Models" are redundant

**Review by 5Ka-me — CHANGES_REQUESTED** [review]

File: API/ViewModels/CreateViewModels/CreateVehicleViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+namespace BLL.ViewModels;
+
+public record CreateVehicleViewModel(
+    Guid ModelId,
+    string PlateNumber,
+    double RentCost,
+    VehicleType VehicleType,
+    VehicleState VehicleState,
+    FuelType FuelType,
+    int Odo = 0
```
Comment (5Ka-me) [inline API/ViewModels/CreateViewModels/CreateVehicleViewModel.cs:12]: What is it?

File: API/ViewModels/CreateViewModels/CreateVehicleViewModel.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿using DAL.Enums;
+
+namespace BLL.ViewModels;
+
+public record CreateVehicleViewModel(
+    Guid ModelId,
+    string PlateNumber,
+    double RentCost,
```
Comment (5Ka-me) [inline API/ViewModels/CreateViewModels/CreateVehicleViewModel.cs:8]: Better to use decimal for money

File: API/ViewModels/VchViewModel.cs [hunk 4]
```diff
@@ -0,0 +1,9 @@
+﻿using DAL.Entities;
+
+namespace BLL.ViewModels;
+public record VchViewModel(
```
Comment (5Ka-me) [inline API/ViewModels/VchViewModel.cs:4]: Add empty line

File: API/ViewModels/VchViewModel.cs [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+    Guid Id,
+    DateTime UseTime,
+    VehicleViewModel? Vehicle,
+    ClientViewModel? Client
+    );
```
Comment (5Ka-me) [inline API/ViewModels/VchViewModel.cs:9]: Maybe add TimeStamp or type of action f.e. "Start renting", "End renting"

File: API/ViewModels/VehicleViewModel.cs [hunk 6]
```diff
@@ -0,0 +1,16 @@
+﻿using DAL.Entities;
+using DAL.Enums;
+
+namespace BLL.ViewModels;
+
+public record VehicleViewModel(
+    Guid Id,
+    string PlateNumber,
+    int Odo,
```
Comment (5Ka-me) [inline API/ViewModels/VehicleViewModel.cs:9]: What is it?

File: API/ViewModels/VehicleViewModel.cs [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+    double RentCost,
```
Comment (5Ka-me) [inline API/ViewModels/VehicleViewModel.cs:10]: Use decimal

**Review by 5Ka-me — CHANGES_REQUESTED** [review]

File: BLL/DI/ServicesConfiguration.cs [hunk 8]
```diff
@@ -11,8 +13,10 @@ public static class ServicesConfiguration
     public static void AddApplicationDependencies(this IServiceCollection services, IConfiguration configuration)
     {
         services.AddDataAccessDependencies(configuration);
-        services.AddCustomServicesDependencies();
-        
+
+        TypeAdapterConfig.GlobalSettings.Scan(Assembly.GetExecutingAssembly());
+
+        services.AddCustomServicesDependencies();
```
Comment (5Ka-me) [inline BLL/DI/ServicesConfiguration.cs:19]: As I said better to rename this method

**Review by 5Ka-me — APPROVED** [review]

**Pull request merged**